src/libZilliqa/Zilliqa.cpp

```diff
@@ -172,6 +172,11 @@ Zilliqa::Zilliqa(const PairOfKey& key, const Peer& peer, SyncType syncType,
     LOG_GENERAL(FATAL, "Archvial lookup is true but not lookup ");
   }
 
+  if (SyncType::NEW_SYNC == syncType) {
+    // Setting it earliest before even p2pcomm is instantiated
+    m_n.m_runFromLate = true;
```

Comment:
Does this mean we can remove line 274?

Reply:
I had preferred to kept it safer because there are other calls in between which might change the syncType.